### PR TITLE
[Datumaro] MOT format

### PR DESCRIPTION
**Pull request by zhiltsov-max**
Added support for MOT detection det and gt formats for video sequences.

### CODE REVIEW

File: datumaro/datumaro/util/__init__.py [hunk 1]
```diff
@@ -17,4 +18,28 @@ def dir_items(path, ext, truncate_ext=False):
             if truncate_ext:
                 f = f[:ext_pos]
             items.append(f)
-    return items
\ No newline at end of file
+    return items
+
+def split_path(path):
```
Comment (nmanovic) [inline datumaro/datumaro/util/__init__.py]: I prefer something like code below:

```
def split_path(p):
    head, tail = os.path.split(p)
    return split_path(head) + [tail] if tail else [head]
```

File: datumaro/tests/test_mot_format.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+        class DstExtractor(Extractor):
+            def __iter__(self):
+                return iter([
+                    DatasetItem(id=1,
+                        image=np.ones((16, 16, 3)),
+                        annotations=[
+                            Bbox(0, 4, 4, 8, label=2, attributes={
+                                'occluded': True,
+                                'visibility': 0.0,
```
Comment (nmanovic) [inline datumaro/tests/test_mot_format.py]: Why here do we have visibility 0.0?

File: datumaro/tests/test_mot_format.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                            Bbox(1, 2, 4, 2, label=3, attributes={
+                                'occluded': False,
+                                'visibility': 1.0,
+                                'ignored': False,
+                            }),
+                        ]
+                    ),
+
+                    DatasetItem(id=3,
+                        image=np.ones((5, 4, 3)) * 3,
```
Comment (nmanovic) [inline datumaro/tests/test_mot_format.py]: Why here we don't have attributes at all?

File: datumaro/tests/test_mot_format.py (hunk 2 above)
Comment (zhiltsov-max) [reply datumaro/tests/test_mot_format.py]: We had an object, which was `occluded`, but it's exact visibility was unknown.

File: datumaro/tests/test_mot_format.py (hunk 3 above)
Comment (zhiltsov-max) [reply datumaro/tests/test_mot_format.py]: `DatasetItem`s do not have `attribute`s yet.

File: datumaro/datumaro/util/__init__.py (hunk 1 above)
Comment (zhiltsov-max) [reply datumaro/datumaro/util/__init__.py]: Nice variant, but it should probably include `normpath` call too, and have some handling for 1-element paths like ".." as it gives ['', '..'].

**Pull request merged**